### DISCUSSION TITLE
Porposal: separate UI and app

### DISCUSSION THREAD

**Discussion by AugustinMauroy** [zane-ops/zane-ops#103] [Ideas]
You should have one package for ui and one for app. It could also be an opportunity to set up a storybook.

**Comment by Fredkiss3** [comment]
We don't have a design system, as of the moment we use shadcn ui but we create pages and components ad-hoc when we need them.

If we had a design maybe, but then again, I've been going more and more simpler in my projects because setting up storybook is annoying and add more friction to me when working on this project, we may add something in the future.

**Comment by AugustinMauroy** [comment]
@Fredkiss3 answers to the question
